src/app/entry/delete/dialog/delete-entry-dialog.component.html

```diff
@@ -0,0 +1,22 @@
+<h1 mat-dialog-title>Are you sure you want to delete this Dockstore {{ term }}?</h1>
+<div mat-dialog-content>
+  <app-alert></app-alert>
+  <p>You are about to permanently delete this {{ term }}. After you delete it, it will no longer appear on Dockstore.</p>
+  <p>A {{ term }} can be deleted if it:</p>
+  <ul>
+    <li>Has never been published,</li>
+    <li>Was created on/after April 1, 2022, and</li>
+    <li>Is not a checked or checker workflow.</li>
+  </ul>
+
+  <p>
+    After you delete the {{ term }}, you must uninstall the Dockstore GitHub App from the source GitHub repo, or edit/remove the
+    .dockstore.yml file so that it no longer describes the {{ term }}. If you do not, the {{ term }} may reappear on Dockstore the next time
+    you push to the repo.
+  </p>
+  <p>Are you sure you want to delete this {{ term }}?</p>
+  <div class="btn-group" fxLayout="row" fxLayoutAlign="end center" fxLayoutGap="1rem">
+    <button mat-flat-button (click)="no()" data-cy="delete-no" [disabled]="clicked" mat-dialog-close class="secondary-1">No</button>
+    <button mat-flat-button (click)="yes()" data-cy="delete-yes" [disabled]="clicked" color="warn">Yes, delete this {{ term }}</button>
```

Comment:
I think we should avoid using `No/Yes` although I think we have a few dialogs that do this on the accounts page. Most of our other dialogs use `Cancel/Verb` and the examples in the [Google Material design ](https://m3.material.io/components/dialogs/guidelines) guide do that too. I suggest `Cancel/Delete this {{term}}` or just `Cancel/Delete`